### DISCUSSION TITLE
2021-20: 西安与破壁机

### DISCUSSION THREAD

**Discussion by Xuanwo** [Xuanwo/blog#218] [Announcements]
https://xuanwo.io/reports/2021-20/ 

 An infrastructure engineer, focused on distributed storage system

**Comment by bestony** [comment]
言之有物的周报/月报其实是需要一些言之无物的日报来支撑的。不然很难写的。

**Comment by Prnyself** [comment]
感觉生活化的东西写一些会让读者感觉你是个有血有肉的人，而不是某个社区里热爱技术的 geek 人设

**Comment by xxchan** [comment]
家用什么场景需要制冰机快速制冰？感觉冰格提前做好够用🤔

**Comment by bokket** [comment]
来西安  找我吃饭～（：

**Comment by Xuanwo** [comment]
> 家用什么场景需要制冰机快速制冰？感觉冰格提前做好够用🤔

感觉每天准备冰格太麻烦了，不如等个几分钟就有冰块。不过实际用起来感觉确实不太好用（
